torchtext/datasets/text_classification.py

```diff
@@ -113,36 +113,88 @@ def get_vocab(self):
         return self._vocab
 
 
-def _setup_datasets(dataset_name, root='.data', ngrams=2, vocab=None, include_unk=False):
```

Comment:
Setting ngrams to 2 might have been a mistake on my end.

I'm not a fan of returning bigrams by default because:
a) the default vocab will be much larger
b) it's more intuitive in my opinion to get lines of tokens instead of tokens + bigram tokens
c) we can expect some users to maybe want to combine this with ngram iterator or similar abstractions, which can lead to nasty bugs

Reply:
Set `ngrams` default value to 1